pandas_sphinx_theme/static/css/custom.css

```diff
@@ -288,4 +295,21 @@ div.prev-next-bottom a.right-next {
 
 div.prev-next-bottom a.right-next:after {
   content: " >>"
-}
\ No newline at end of file
+}
+
+/* Social media buttons */
+
+i.fab:before {
+  font-family: 'FontAwesome';
+  vertical-align: middle;
+  font-style: normal;
+  font-size: 1.5rem;
+}
+
+i.fa-github-square:before {
+  content: "\f092";
```

Comment:
For the font-awesome noob: what is this doing?

Reply:
this is the unicode version of a fontawesome icon. FontAwesome is a standard icon pack that you can load via CDN and then use for icons on your site. The Bokeh docs are using fontawesome for their github / twitter logos so I figured we can just copy the same pattern here